totalContributors.py

```diff
@@ -0,0 +1,83 @@
+import subprocess
+import pathlib
+import os.path
+import sys
+import datetime
+
+def add_log(dict, input):
+    for dataset in input:
+        name = dataset.split("\t")[1]
+        value = int(dataset.split("\t")[0])
+        if name in dict:
+            dict[name] += value
+        else:
+            dict[name] = value
+
+def retrieve_log(repo, version):
+    return subprocess.run(['git', 'shortlog', '-s', version], capture_output = True, text = True, 
+        cwd = os.path.dirname(pathlib.Path().resolve()) + repo).stdout.splitlines()
+
+def find_version(time, repo):
+    versions_list = subprocess.run(['git', 'tag', '-l'], capture_output = True, text = True, 
```

Comment:
I usually see `subprocess.check_output` in the repo, e.g. https://github.com/zulip/zulip/blob/master/scripts/lib/setup_venv.py#L181. And so preferrable you should use `check_output` so that the code is uniform.